packages/react/vite.config.ts

```diff
@@ -50,7 +55,10 @@ export default defineConfig(({ mode }) => {
             // Move index.css to root of dist
             `mv dist/esm/index.css dist/index.css`,
             // Delete extra .css.js files
-            `rm dist/**/*.css.*`,
+            `find ./dist -name "*.css.js" -delete`,
```

Comment:
Nit: I think we could get this to a single line catching anything `.css.*`: ``find ./dist -name "*.css.*" -delete``